charmcraft/commands/store/__init__.py

```diff
@@ -46,9 +46,20 @@
 if TYPE_CHECKING:
     from argparse import ArgumentParser, Namespace
 
+
 # some types
-EntityType = namedtuple("EntityType", "charm bundle")(charm="charm", bundle="bundle")
-ResourceType = namedtuple("ResourceType", "file oci_image")(file="file", oci_image="oci-image")
+class _EntityType(typing.NamedTuple):
```

Comment:
I did these NamedTuple changes manually... I chose to do the minimal change to make the linters happy without #noqa since this will change very soon.